inventory/sample/group_vars/k8s-cluster.yml

```diff
@@ -251,3 +251,16 @@ persistent_volumes_enabled: false
 ## See https://github.com/kubernetes-incubator/kubespray/issues/2141
 ## Set this variable to true to get rid of this issue
 volume_cross_zone_attachment: false
+
+## Container Engine Acceleration
+## Enable container accelartion feature, for example use gpu acceleration in containers
+container_engine_acceleration_enabled: false
+## Nvidia GPU driver install. Install will by done by a (init) pod running as a daemonset.
+## Array with nvida_gpu_nodes, leave empty or comment if you dont't want to install drivers.
+## Nodes won't get labels gpu labels if they are not in the array.
+## Important: this should be set in all.yml 'docker_storage_options: -s overlay2'
```

Comment:
Is the docker storage options actually relevant to GPU installation?

Reply:
yes, because the ubuntu nvidia dirver installer image user overlay mounts. If the storage option is not set to `overlay2` ubuntu will use `aufs` and the driver installer will result is a kernel panic.